python/lsst/ap/verify/pipeline_driver.py

```diff
@@ -66,98 +46,50 @@ class ApPipeParser(argparse.ArgumentParser):
     def __init__(self):
         # Help and documentation will be handled by main program's parser
         argparse.ArgumentParser.__init__(self, add_help=False)
-        self.add_argument('--id', dest='dataId', required=True,
-                          help='An identifier for the data to process. '
-                          'May not support all features of a Butler dataId; '
-                          'see the ap_pipe documentation for details.')
+        self.add_argument('--id', dest='dataId', default="",
+                          help='An identifier for the data to process.')
         self.add_argument("-j", "--processes", default=1, type=int,
                           help="Number of processes to use. Not yet implemented.")
```

Comment:
This is what I was basing my "optimism" comment on; your statement here that multiprocessing (i.e., with the -j keyword) is not yet implemented. If it is implemented now that's even better, and please update this docstring!

Reply:
I could swear I fixed that. Good catch, thanks!